recipe/meta.yaml

```diff
@@ -1,45 +1,54 @@
-{% set name = "<PLEASE ADD PKG NAME>" %}
-{% set version = "<PLEASE ADD PKG VERSION>" %}
+{% set name = "conda-anaconda-telemetry" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "01044e9d7cc51eab3fa70e0ffa0b7638e20c90ce00e1b346e97d94288d1111fc" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: <sha256>
+  url: https://github.com/anaconda/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  skip: True  # [py<38]
+  noarch: python
```

Comment:
@jezdez Can you add a comment above to explain why we use `noarch: python` here? This would help the next time we look at this recipe and avoid unnecessary regressions.